### PR TITLE
fix: Add error for multiple read modes

### PR DESCRIPTION
**Pull request by nathan-stender**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Handle comma as decimal place in float conversion utilities.
+- Add error for multiple read modes in aglient gen5
```
Comment (mdlu) [inline CHANGELOG.md:17]: nit: should probably capitalize Agilent Gen5

File: src/allotropy/parsers/agilent_gen5/constants.py [hunk 2]
```diff
@@ -45,6 +46,11 @@ class ReadMode(str, Enum):
     ALPHALISA = "Alpha"
 
 
+UNSUPORTED_READ_MODE_ERROR = (
```
Comment (mdlu) [inline src/allotropy/parsers/agilent_gen5/constants.py:49]: should be UNSUPPORTED

File: src/allotropy/parsers/agilent_gen5/constants.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 from enum import Enum
 
 MULTIPLATE_FILE_ERROR = "Only a single plate per file can be processed at this time. Please refer to Gen5 documentation for how to generate single plate exports from multi-plate experiments"
+MULTIPLE_READ_MODE_ERROR = "Only a single read mode per file can be processed at this time. Please refer to Gen5 documentation for how to generate single mode exports from multi-mode experiments"
 NO_PLATE_DATA_ERROR = "No plate data found in file."
 UNSUPORTED_READ_TYPE_ERROR = "Only Endpoint measurements can be processed at this time."
```
Comment (mdlu) [inline src/allotropy/parsers/agilent_gen5/constants.py:6]: should be UNSUPPORTED

File: tests/parsers/agilent_gen5/to_allotrope_test.py [hunk 4]
```diff
@@ -64,6 +65,12 @@ def test_to_allotrope_unsupported_area_scan_file() -> None:
         from_file(filepath, VENDOR_TYPE)
 
 
+def test_to_allotrope_unsupported_multiple_read_modes() -> None:
+    filepath = f"{TESTDATA}/multiple_read_modes_error.txt"
```
Comment (mdlu) [inline tests/parsers/agilent_gen5/to_allotrope_test.py:69]: where is this file?

Comment (nathan-stender) [reply tests/parsers/agilent_gen5/to_allotrope_test.py:69]: Oops, added

**Review by mdlu — APPROVED** [review]

**Pull request merged**